### DISCUSSION TITLE
The datanode remains in the Health status for long time after being turned off

### DISCUSSION THREAD

**Discussion by unixsrv** [apache/ozone#7527] [FAQ] [answered]
Please suggest which parameters to specify for the Datanode output to be fast and smooth, as currently after turning off the datanode, it remains in a health state while the active copy task hangs.

**Comment by unixsrv** [comment]
Resolved

**Reply by errose28** [reply]
Could you share how you resolved the issue for others who may have a similar question? Were you looking for the configurations `ozone.scm.stale.node.interval` (defaults to 5 minutes, after which node is not used for writes) or `ozone.scm.dead.node.interval` (defaults to 10 minutes, after which replication would be triggered)? Did you cluster have enough nodes to service the data write when one is taken offline (for example, starting with at least 4 nodes so that 3-way replicated data can be written if one goes down)?

**Reply by unixsrv** [reply, marked as answer]
Yes, of course. The cluster consists of 4 datanodes, with a RATIS/THREE pipeline. As you recommended above, the parameters were changed:
```
    <property>
      <name>ozone.scm.stale.node.interval</name>
      <value>5s</value>
    </property>
    <property>
      <name>ozone.scm.dead.node.interval</name>
      <value>45s</value>
    </property>
    <property>
        <name>hdds.heartbeat.interval</name>
        <value>10s</value>
    </property>
    <property>
      <name>hdds.node.report.interval</name>
      <value>20s</value>
    </property>
```
